openmmforcefields/tests/test_template_generators.py

```diff
@@ -1,3 +1,4 @@
+# ruff: noqa: E501
```

Comment:
@mattwthompson yes I don't think it a bad thing, so I just had it skip this whole file and this was the only file where they were too long

Reply:
Seems good, the config is still pretty simple